nci/lib/setup_repo.rb

```diff
@@ -59,8 +59,11 @@ def setup_repo!(with_source: false, with_proxy: true, with_install: true)
     # Make sure we have the latest pkg-kde-tools, not whatever is in the image.
     return unless with_install
 
-    raise 'failed to install deps' unless Apt.install(%w[pkg-kde-tools pkg-kde-tools-neon debhelper cmake quilt dh-python dh-translations kde-release-keyring])
-
+    @deps = %w[pkg-kde-tools pkg-kde-tools-neon debhelper cmake quilt dh-python dh-translations]
```

Comment:
this should be `deps` not `@deps`.

`@` denotes an instance variable. i.e. it is available in the entire Object, which isn't necessary here. what we need is a variable that is limited to the local scope

Reply:
thanks for your patience ...